sprintQuovantis/sprintParam/views.py

```diff
@@ -37,42 +43,117 @@ def post(self, request):
                 status=status.HTTP_400_BAD_REQUEST)
         except Exception as e:
             logging.error(e)
+            print(e)
```

Comment:
For this generic exception, response code should be 500 i.e. internal server error. That's because we didn't get an exception from the the serializer validation, something else went wrong instead. Returning 400 bad request body would be misleading.